File: create_dir.py
#!/usr/bin/env python3
import os
import sys

target_path = r'c:\Users\marlon.villarama\OneDrive - Datacom\Documents\DEV\LAB\sc-ai-002\src\FileCabinet\SuiteScripts\Collections'

try:
    os.makedirs(target_path, exist_ok=True)
    print(f'✓ Directory created successfully')
    print(f'Path: {target_path}')
    
    if os.path.isdir(target_path):
        print('✓ Directory verified - it exists')
    else:
        print('✗ Directory verification failed')
        sys.exit(1)
except Exception as err:
    print(f'Error creating directory: {err}')
    sys.exit(1)

File: create_dir.js
const fs = require('fs');
const path = require('path');

const targetPath = 'c:\\Users\\marlon.villarama\\OneDrive - Datacom\\Documents\\DEV\\LAB\\sc-ai-002\\src\\FileCabinet\\SuiteScripts\\Collections';

try {
  fs.mkdirSync(targetPath, { recursive: true });
  console.log('✓ Directory created successfully');
  console.log(`Path: ${targetPath}`);
  
  if (fs.existsSync(targetPath)) {
    console.log('✓ Directory verified - it exists');
  }
} catch (err) {
  console.error('Error creating directory:', err.message);
  process.exit(1);
}
